bitnami/solr/README.md

```diff
@@ -0,0 +1,356 @@
+| `cloudEnabled`                         | Enable Solr cloud mode                                                                           | `true`                             |
+| `cloudBootstrap`                       | Bootstrap the Solr cloud cluster on the install                                                  | `true`                             |
+| `collection`                           | Name of the collection to be created                                                             | `my-collection`                    |
+| `collectionShards`                     | Number of collection shards                                                                      | `1`                                |
+| `collectionReplicas`                   | Number of collection replicas                                                                    | `2`                                |
+| `containerPort`                        | Port number where Solr is running inside the container                                           | `8983`                             |
+| `serverDirectory`                      | Name of the created directory for the server                                                     | `server`                           |
+| `javaMem`                              | Java memory options to pass to the Solr container                                                | `nil`                              |
+| `heap`                                 | Java Heap options to pass to the solr container                                                  | `nil`                              |
+| `authentication.enabled`               | Enable Solr authentication                                                                       | `true`                             |
+| `authentication.adminUsername`         | Solr admin username                                                                              | `admin`                            |
+| `authentication.adminPassword`         | Solr admin password. Autogenerated if not provided.                                              | `mil`                              |
+| `existingSecret`                       | Existing secret with Solr password                                                               | `nil`                              |
+| `command`                              | Override Solr entrypoint string.                                                                 | `nil`                              |
+| `args`                                 | Arguments for the provided command if needed                                                     | `nil`                              |
+| `podAffinityPreset`                    | Solr pod affinity preset. Ignored if `affinity` is set. Allowed values: `soft` or `hard`         | `""`                               |
```

Comment:
podLabels, podAnnotations, lifecycleHooks, persistence.annotations and persistence.mountPath are missing.


Double check this list and ignore the false positives
```
ERROR: Value service.nodePorts.http is not in README.md
ERROR: Value service.nodePorts.https is not in README.md
ERROR: Value podLabels is not in README.md
ERROR: Value podAnnotations is not in README.md
ERROR: Value lifecycleHooks is not in README.md
ERROR: Value volumePermissions.containerSecurityContext is not in README.md
ERROR: Value persistence.annotations is not in README.md
ERROR: Value persistence.mountPath is not in README.md
ERROR: Value zookeeper.service.publishNotReadyAddresses is not in README.md
ERROR: Value exporter.updateStrategy is not in README.md
ERROR: Value exporter.service.nodePorts.http is not in README.md
ERROR: Value exporter.service.nodePorts.https is not in README.md
ERROR: Value exporter.service.loadBalancerSourceRanges is not in README.md
```